website/docs/d/is_security_group_rule.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_is_security_group_rule"
+description: |-
+  Get information about is_security_group_rule
+subcategory: "Virtual Private Cloud API"
+---
+
+# ibm_is_security_group_rule
+
+Provides a read-only data source for is_security_group_rule. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_is_security_group_rule" "example" {
+	id = ibm_is_security_group_rule.testacc_security_group_rule_all.id
+	security_group_id = ibm_is_security_group.testacc_security_group.id
+}
+
+data "ibm_is_security_group_rule" "example" {
+	id = ibm_is_security_group_rule.testacc_security_group_rule_udp.id
```

Comment:
references in the syntax should also follow "example" style !?

Reply:
done.